packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,67 @@
+import { attr, observable } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorsScale,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'quadrant'
+    })
+    public quadrant: WaferMapQuadrant = WaferMapQuadrant.topLeft;
+
+    @attr({
+        attribute: 'orientation'
+    })
```

Comment:
since the attribute and property name align, this can be simplified to
```suggestion
    @attr
```